spec/factories/plate_factories.rb

```diff
@@ -327,7 +327,7 @@
   end
 
   # Basic API v1 plate
-  factory :plate, class: Limber::Plate, traits: %i[api_object barcoded] do
+  factory :plate, class: Plate, traits: %i[api_object barcoded] do
```

Comment:
The comment says `Basic API v1 plate`...

Are we still using this factory, and similar ones (e.g. qcable, below)? Maybe to be removed  in a subsequent PR?